.mlc_config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "aliveStatusCodes": [
+    403,
+    200,
+    206
+  ],
+  "httpHeaders": [
+    {
+      "urls": [
+        "https://github.com/",
+        "https://guides.github.com/",
+        "https://help.github.com/",
+        "https://docs.github.com/"
+      ],
+      "headers": {
+        "Accept-Encoding": "zstd, br, gzip, deflate"
+      }
+    }
+  ],
+  "retryOn429": true,
+  "ignorePatterns": [
+    {
+      "pattern": "/github/workspace/.*"
+    },
+    {
+      "pattern": "crates.io"
+    }
+  ]
```

Comment:
Don't check links with theses patterns/regexes:

1. `"/github/workspace/.*"`: any internal links in the book of bdk
2. `"crates.io"`: is not friendly to markdown link checker, so we are disabling for now.